### ISSUE TITLE
hammerhead.js __get$(document, "location").ancestorOrigins returns <unreadable>

### ISSUE THREAD

**Issue by ssemrau** [DevExpress/testcafe#7022]
### What is your Scenario?

Fuse Microform iframes should be shown in given div containers.
See this doc: https://developer.cybersource.com/docs/cybs/en-us/digital-accept-flex/developer/all/rest/digital-accept-flex/microform-integ/microform-integ-getting-started/setting-up-client-side.html

### What is the Current behavior?

When I test it manually all works fine, when I run the test script I get the issue that `__get$(document, "location").ancestorOrigins` returns value unreadable as per screenshot below
![1](https://user-images.githubusercontent.com/11912960/167857174-61eab5a6-8872-47cb-b1d3-3ed57d55988a.png)

### What is the Expected behavior?

`__get$(document, "location").ancestorOrigins` returns 'http://localhost:12345'

### What is your public website URL? (or attach your complete example)

https://flex-microform-hammerhead-issue.azurewebsites.net/

### What is your TestCafe test code?

```
import { Selector } from 'testcafe';

fixture('Azure')
    .page('about:blank');

test('Azure', async t => {    
    await t
        .maximizeWindow()        
        .navigateTo('https://flex-microform-hammerhead-issue.azurewebsites.net/');

    var mainWindow = await t.getCurrentWindow() 

    try {
        await t
            .typeText('#cardholderName', 'Tester')
            .wait(1000);
        
        await t.debug();

        await t
            .switchToIframe('#number-container > iframe')
            .typeText('[name="number"]', '4111111111111111')
            .switchToWindow(mainWindow);
        
        await t
            .switchToIframe(`#securityCode-container > iframe`)
            .typeText('[name="securityCode"]', '123')
            .switchToWindow(mainWindow);

        await t
            .click('#pay-button');    

        await t
            .wait(1000)
            .click('#pay-button');    
    } catch (ex) {
        console.error(ex);
    }

    await t.debug();
});
```

### Your complete configuration file

.testcaferc.json
`{
    "skipJsErrors": true,
    "hostname": "localhost",
    "port1": 12345,
    "port2": 12346
}`

### Screenshots

![1](https://user-images.githubusercontent.com/11912960/167855817-851b458c-5fa7-43ed-b9b5-24dd9e816b66.png)
![3](https://user-images.githubusercontent.com/11912960/167855851-30739a20-5b92-4fac-8008-0aa7f6ba2f89.png)
![2](https://user-images.githubusercontent.com/11912960/167855838-cace988a-ea20-4672-84ed-707e0ae48a4c.png)

### Steps to Reproduce

1. Run automated script in chrome
2. When script stops open dev tools
3. Switch to Console
4. Unclock the page and press `Mount Security Code` button
5. Expected error should be 
```
correlationId: undefined
details: undefined
informationLink: "https://www.cybersource.com/products/payment_security/secure_acceptance"
message: "You have not supplied a valid capture context."
name: "MicroformError"
reason: "CAPTURE_CONTEXT_INVALID"
```
6. Open iframe.min.js and format it it so that it's not minified, add breakpoint in line 2772 as per screenshot below
![4](https://user-images.githubusercontent.com/11912960/167860987-77eb0fc6-b7e0-4813-a19b-2d6fbfb571e3.png)
7. Press `Mount Security Code` button again and in debug inspect `document` and `__get$(document, "location")` variables  

### TestCafe version

1.18.6

### Node.js version

v14.17.6

### Command-line arguments

testcafe chrome azure.js

### Browser name(s) and version(s)

Chrome 101.0.4951.54 / Windows 10

### Platform(s) and version(s)

Windows

**Comment by Aleksey28** [comment]
Hi @ssemrau,

Thank you for your example. I reproduced the issue. Please share your project with the code for further research.

**Comment by ssemrau** [comment]
> Hi @ssemrau,
> 
> Thank you for your example. I reproduced the issue. Please share your project with the code for further research.

Hi @Aleksey28,
what would you like me to share? Code of the application? Code of Microform?

**Comment by Aleksey28** [comment]
Please share the code of your application where you use this microform. I need a complete example that I can use to reproduce the issue locally without complex settings. This will speed up our research.

**Comment by ssemrau** [comment]
There is github repository provided by Cybersource which is also provider of Microform
I used this https://github.com/CyberSource/cybersource-flex-samples-dotnet/tree/master/flex-microform-sample as example. It's in .net 4.6 but there are as well node.js examples which are very similar.
I tried as well run testcafe 1.13 and it seems Microform works better, it fails sometimes, but it's more stable. Just a hint.
Of course downgrading to 1.13.0 is not a solution as we would like to use the latest version of testing framework.
